pkg/network/tracer/connection/perf_batching.go

```diff
@@ -10,14 +10,13 @@ package connection
 import (
 	"fmt"
 	"time"
-	"unsafe"
 
-	"github.com/cilium/ebpf"
+	manager "github.com/DataDog/ebpf-manager"
```

Comment:
nit: this should be grouped with other DataDog/ imports below.

Reply:
Tried to, but the automatic import sorter doesn't seem to like it :(

Reply:
imports from outside `github.com/DataDog/datadog-agent` do not get grouped with them

Reply:
I thought we were supposed to group `github.com/DataDog` imports together

Reply:
https://github.com/DataDog/datadog-agent/blob/main/docs/dev/imports.md

`github.com/DataDog/ebpf-manager` is considered external to the `github.com/DataDog/datadog-agent` repo.